### PR TITLE
chore: add cpu time axis title

### PR DESCRIPTION
**Pull request by shhdgit**
- Add CPU time axis title

image

### CODE REVIEW

File: ui/lib/apps/TopSQL/translations/zh.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ topsql:
     others: 其他语句
     others_tooltip: 所有其他非 Top {{topN}} 的语句
     fields:
-      cpu_time: CPU
+      cpu_time: CPU Time
```
[the remark this answers is not in the document]
Comment (shhdgit) [reply ui/lib/apps/TopSQL/translations/zh.yaml]: done!

File: ui/lib/apps/TopSQL/pages/List/ListChart.tsx [hunk 2]
```diff
@@ -79,6 +79,7 @@ export const ListChart = forwardRef<Chart, ListChartProps>(
         />
         <Axis
           id="left"
+          title="CPU Time"
```
Comment (breeswish) [inline ui/lib/apps/TopSQL/pages/List/ListChart.tsx:82]: Seems that this is not calling i18n functions. Will it cause problems?

Comment (shhdgit) [reply ui/lib/apps/TopSQL/pages/List/ListChart.tsx:82]: That's right! Using i18n will be better.

**Comment by shhdgit** [comment]
/cc @breeswish @baurine 
PTAL, thanks~

**Review by breeswish — APPROVED** [review]

**Pull request merged**